### DISCUSSION TITLE
我想问几个问题。CNAUTO开着，但是CUSTOM_FORWARD不写会怎么样

### DISCUSSION THREAD

**Discussion by zkf0802** [kkkgo/PaoPaoDNS#157] [问答和帮助]
我想问几个问题。
1、CNAUTO开着，但是CUSTOM_FORWARD不写会怎么样？
2、我是不是可以搭建一个分流ppdns，再搭建一个纯粹递归的ppdns，在分流ppdns上CUSTOM_FORWARD指定为纯粹的ppdns，这样实现无污染的dns系统？

**Comment by kkkgo** [comment]
CNAUTO本来就是分流无污染的，CUSTOM_FORWARD的用途一般是FAKEIP相关的，不是你想的功能。

**Reply by zkf0802** [reply]
所以不能单独用两个ppdns搭建一个全球无污染的dns是吗？

**Reply by kkkgo** [reply]
CNAUTO本来就是分流无污染的

**Reply by zkf0802** [reply]
你的意思是CNAUTO开了之后，就可以全球的无污染，根本就不需要第二个ppdns对吧，CUSTOM_FORWARD可以不设置。

**Reply by kkkgo** [reply]
是的，也可以设置SOCKS5变量，效果更好。
CUSTOM_FORWARD的用途一般用于FAKEIP透明代理，可以参考：https://github.com/kkkgo/PaoPaoGateWay/discussions/23
